cli/tests/cli-reference@.md.snap

```diff
@@ -1511,8 +1511,66 @@ Note that you can create a merge commit by specifying multiple revisions as argu
 * `-m`, `--message <MESSAGE>` — The change description to use
 * `--no-edit` — Do not edit the newly created change
 * `-A`, `--insert-after <REVSETS>` — Insert the new change after the given commit(s)
+
+   Example: `jj new -A 1` creates a new change between `1` and its
+   children:
+
+   ```text
+
+                 2   3
+                  \ /
+     2   3   =>    @
+      \ /          |
+       1           1
+
+   ```
+
+   Specifying `-A` multiple times will relocate all children of the given
+   commits.
+
+   Example: `jj new -A 1 -A 3` creates a change with `1` and `3` as
+   parents, and rebases all children on top of the new change:
+
+   ```text
+                 2   4
+                  \ /
+     2  4   =>     @
+     |  |         / \
+     1  3        1   3
+
+   ```
 * `-B`, `--insert-before <REVSETS>` — Insert the new change before the given commit(s)
 
+   Example: `jj new -B 3` creates a new change between `3` and its parents:
+
+   ```text
+
+                    3
+                    |
+       3     =>     @
+      / \          / \
+     1   2        1   2
+
+   ```
+
+   `-A` and `-B` can be combined, which will limit which commits are
+   rebased.
+
+   Example: `jj new -A 1 -B 2` creates a change between `1` and `2`,
+   but does not touch `3`:
+
+   ```text
+
+                  2
+                  |
+     2   3   =>   @   3
+      \ /          \ /
+       1            1
```

Comment:
`jj new -B 2` would have been sufficient in this case, so I think we should give another example where both the `-A` and the `-B` are needed.

Reply:
Done, I changed it to a stitching thing

```rust
    /// Example: `jj new --after A --before D`:
    ///
    /// ```text
    /// 
    ///     D            D
    ///     |           / \
    ///     C          |   C
    ///     |    =>    @   |
    ///     B          |   B
    ///     |           \ /
    ///     A            A
    /// ```
```

Reply:
(empty line b/c otherwise it glues to "text" unpleasantly)